src/app/shared/Note.svelte

```diff
@@ -74,7 +75,12 @@
 
   const addDraftToContext = (event, cb) => {
     draftEventId = event.id
-    removeDraft = () => cb() && repository.removeEvent(event.id)
+    removeDraft = () => {
+      cb()
+      repository.removeEvent(event.id)
```

Comment:
If you're using `publishThunk`, this shouldn't be necessary, welshman does it automatically.